sql/migrations/20201010150010_world.sql

```diff
@@ -0,0 +1,123 @@
+(128403, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -47, 0, 0, 0, 0, 0, 0, 0, 0, 'Archbishop Benedictus - Say Text');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (128403, 1284, 0, 2, 0, 100, 0, 15, 0, 0, 0, 128403, 0, 0, 'Archbishop Benedictus - Flee at 15% HP');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(128404, 0, 15, 20697, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Archbishop Benedictus - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (128404, 1284, 0, 0, 0, 100, 1, 12000, 18000, 35000, 45000, 128404, 0, 0, 'Archbishop Benedictus - Cast Power Word: Shield');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(128405, 0, 15, 20694, 0, 0, 0, 0, 0, 17, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Archbishop Benedictus - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (128405, 1284, 0, 9, 0, 100, 1, 0, 5, 12000, 19000, 128405, 0, 0, 'Archbishop Benedictus - Cast Holy Nova');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(128406, 0, 15, 20695, 8, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Archbishop Benedictus - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (128406, 1284, 0, 9, 0, 100, 1, 0, 40, 3400, 4800, 128406, 0, 0, 'Archbishop Benedictus - Cast Holy Smite');
+UPDATE `creature_template` SET `ai_name` = 'EventAI' WHERE `entry` = 1284;
+
+
+-- Lady Jaina Proudmoore 4968

```

Comment:
Jaina has a c++ script and uses all these abilities from there. If you want to move it to DB, that's okay, but simple timed casts belong in a spell list, not in Event AI. Do not blindly copy scripts from cmangos. You even left the comments the same, even though we have no such thing as "Ranged Mode". We have a flag on spells inside spell lists, that makes creatures stop moving and melee attacking until the spell cast fails, which produces the same behavior in game. Also if you move her script to db, you will need to hack spell 20682 to remove teleported players from combat, or Jaina will try to chase them all the way to the docks which will cause her to reset.